custom-instrumentation/README.md

```diff
@@ -1,2 +1,12 @@
-## Custom instrumentation
+## Custom Instrumentation
+
 This folder contains example applications using the [Agent API](https://newrelic.github.io/node-newrelic/API.html) to do custom instrumentation.
+
+### Table of Contents
+
+| Directory               | App Name                                    | Purpose                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+| ----------------------- | ------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
this structure differs from others the TOC I have PRd [here](https://github.com/newrelic/newrelic-node-examples/pull/274).